tools/image-tag

```diff
@@ -5,9 +5,9 @@ set -o nounset
 set -o pipefail
 
 WIP=$(git diff --quiet || echo '-WIP')
-BRANCH=$(git rev-parse --abbrev-ref HEAD | sed s#/#-#g)
-SHA=$(git rev-parse --short=7 HEAD | head -c7)
```

Comment:
When 7 chars are not enough to be unique, git automatically uses more. We are forcing to 7 here, as we are doing for grafana/grafana as well.

Reply:
Get it. Add comment here is better i think.
Still perfer to add quota for sed.